tools/jnimarshalmethod-gen/App.cs

```diff
@@ -463,11 +512,11 @@ static TypeDefinition FindType (Type type)
 		}
 	}
 
-	static class Extensions
+	internal static class Extensions
 	{
 		public static string GetCecilName (this Type type)
 		{
-			return type.FullName.Replace ('+', '/');
+			return type.FullName?.Replace ('+', '/');
```

Comment:
My guess is this can now return `null`? If so do we know where this method is called? Do those calling points need to be updated to handle a `null`?

Reply:
The updated path with *null* goes only thru `CompareTypes` where it is used only for comparison.